packages/migrate/src/__tests__/Baseline.test.ts

```diff
@@ -0,0 +1,115 @@
+import { jestConsoleContext, jestContext } from '@prisma/internals'
+import fs from 'fs-jetpack'
+import prompt from 'prompts'
+
+import { DbPull } from '../commands/DbPull'
+import { MigrateDeploy } from '../commands/MigrateDeploy'
+import { MigrateDev } from '../commands/MigrateDev'
+import { MigrateResolve } from '../commands/MigrateResolve'
+
+const ctx = jestContext.new().add(jestConsoleContext()).assemble()
+
+// Disable prompts
+process.env.GITHUB_ACTIONS = '1'
+// Disable generate
+process.env.PRISMA_MIGRATE_SKIP_GENERATE = '1'
+
+// Covered in docs: https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/add-prisma-migrate-to-a-project
+// We have a dev and prod database
+// And we want to baseline the production database with a baseline migration
+// For that, we use `db pull`, `migrate dev`, `migrate resolve` and `migrate deploy` commands
+describe('Baselining', () => {
+  it('SQLite: should succeed', async () => {
+    ctx.fixture('baseline-sqlite')
+    fs.remove('prisma/migrations')
+    fs.copy('prisma/dev.db', 'prisma/prod.db')
+
+    // Start with the dev database
+    process.env.DATABASE_URL = 'file:./dev.db'
```

Comment:
We should always restore the `process.env` global object as we have found it. Maybe use something like

```typescript
  // backup env vars
  const OLD_ENV = { ...process.env }

  afterEach(() => {
    // reset env vars to backup state
    process.env = { ...OLD_ENV }
  })
```

as used elsewhere in the test codebase?

Reply:
I'm wondering whether it wouldn't make even more sense to spawn a CLI as a separate process for each test — mocking and mutating the environment are both going to cause headaches, doubly so as soon as concurrency is involved.

Reply:
But that's probably out of scope for this PR.

Reply:
It's a bit useless because there is only one test though